docs_source_files/content/webdriver/keyboard.de.md

```diff
@@ -3,19 +3,16 @@ title: "Tastatur"
 weight: 10
 ---
 
-{{% notice info %}}
-<i class="fas fa-language"></i> Diese Seite wird von Englisch 
-auf Deutsch übersetzt. Sprichst Du Deutsch? Hilf uns die Seite 
-zu übersetzen indem Du uns einen Pull Reqeust schickst!
- {{% /notice %}}
-Keyboard represents a KeyBoard event. KeyBoard actions are performed by using low-level
-interface which allows us to provide virtualized device input to the web browser.
+Tastatur repräsentiert Tastaturaktionen. Tastatur "Actions" werden mit Hilfe
+eines low-level Interface durchgeführt, dass ein virtualisiertes Eingabegerät
+für den Webbrowser zur Verfügung stellt.
```

Comment:
can we have space in 'Webbrowser'

Reply:
Webbrowser is correct as it is in German (without space)


See here: https://www.duden.de/rechtschreibung/Webbrowser